python-project-template/.github/workflows/{% if include_benchmarks %}asv-pr.yml{% endif %}.jinja

```diff
@@ -1,101 +1,86 @@
+    - name: Set up Python ${{ env.PYTHON_VERSION }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ env.PYTHON_VERSION }}
   asv-pr:
     runs-on: ubuntu-latest
     needs: setup-python
-
-    permissions:
-      actions: read
-      pull-requests: write
-
     defaults:
       run:
-        working-directory: {% raw %}${{ env.WORKING_DIR }}{% endraw %}
-
+        working-directory: ${{ env.WORKING_DIR }}
     steps:
-      - name: Checkout PR branch of the repository
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-
-      - name: Cache Python {% raw %}${{ env.PYTHON_VERSION }}{% endraw %}
-        uses: actions/cache@v3
-        with:
-          path: ~/.cache/pip
-          key: python-{% raw %}${{ env.PYTHON_VERSION }}{% endraw %}
-
-      - name: Install dependencies
-        run: |
-          sudo apt-get update
-          python -m pip install --upgrade pip
-          pip install asv==0.6.1 virtualenv tabulate lf-asv-formatter
-
-      - name: Get current job logs URL
-        uses: Tiryoh/gha-jobid-action@v0
-        id: jobs
-        with:
-          github_token: {% raw %}${{ secrets.GITHUB_TOKEN }}{% endraw %}
-          job_name: {% raw %}${{ github.job }}{% endraw %}
-
-      - name: Create ASV machine config file
-        run: asv machine --machine gh-runner --yes
-
-      - name: Run comparison of PR against main branch
-        run: |
-          git remote add upstream https://github.com/{% raw %}${{ github.repository }}{% endraw %}.git
-          git fetch upstream
-          asv continuous upstream/main HEAD --verbose || true
-          asv compare upstream/main HEAD --sort ratio --verbose | tee output
-          python -m lf_asv_formatter --asv_version "$(echo asv --version)"
-          printf "\n\nClick [here]($STEP_URL) to view all benchmarks." >> output
-        env:
-          STEP_URL: "{% raw %}${{ steps.jobs.outputs.html_url }}{% endraw %}#step:8:1"
-
-      - name: Find benchmarks comment
-        uses: peter-evans/find-comment@v2
-        id: find-comment
-        with:
-          issue-number: {% raw %}${{ github.event.pull_request.number }}{% endraw %}
-          comment-author: 'github-actions[bot]'
-          body-includes: view all benchmarks
-
-      - name: Create or update benchmarks comment
-        uses: peter-evans/create-or-update-comment@v3
-        with:
-          comment-id: {% raw %}${{ steps.find-comment.outputs.comment-id }}{% endraw %}
-          issue-number: {% raw %}${{ github.event.pull_request.number }}{% endraw %}
-          body-path: {% raw %}${{ env.WORKING_DIR }}/output{% endraw %}
-          edit-mode: replace
\ No newline at end of file
+    - name: Checkout PR branch of the repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Display Workflow Run Information
+      run: |
+        echo "Workflow Run ID: ${{ github.run_id }}"
+    - name: Cache Python ${{ env.PYTHON_VERSION }}
```

Comment:
I don't know enough to really know what's happening here, but are we caching Python twice? Here and on line 25-ish? Or is this reading from the cache?

Reply:
@drewoldag you're right, can we address it in another PR? We have an open issue about it https://github.com/lincc-frameworks/python-project-template/issues/259#issuecomment-1674967400.

Reply:
Yep, that's fine with me!